oeps/processes/oep-0060-proc-sec-group.rst

```diff
@@ -0,0 +1,243 @@
+
+Maintainers should inform the Security Working Group if they judge a vulnerability to be a different severity than what was originally triaged. The Security Working Group must accept the maintainer's adjudication, but should comment on any considerations around the adjudication.
+
+Maintainers will be routinely reminded to remediate disclosures in a frequency proportional to the severity of the disclosure. The following table shows the default reminder frequency until resolution of each severity classification:
+
+========    =====   ==================
+Severity    Score   Reminder frequency
+========    =====   ==================
+Low         ≥0.1    Twice a year
+Medium      ≥4.0    Once a quarter
+High        ≥7.0    Once a month
+Critical    ≥9.0    Once a week
+========    =====   ==================
+
+The on-call Security Working Group member is responsible for sending reminders for any vulnerability reminders due during the time they are on-call. These routine reminders should be batched so that all reminders of the same frequency for a maintainer are compiled into a single notification.
+
+.. note::
+
+    Maintainers are encouraged to reach out to security@openedx.org if they need any help remediating security issues.
+
+.. _GitHub repository security advisory: https://docs.github.com/en/code-security/security-advisories/repository-security-advisories/creating-a-repository-security-advisory
+
+Security Vulnerability Privacy
+------------------------------
+Details of security vulnerabilities will remain private until a fix is released. Members of the Open edX community with access to the details of a disclosure are asked to keep those details private until a fix is publicly released for the vulnerability.
+
+Maintainers and their delegates should use GitHub's `temporary private fork`_ feature within the GitHub repository security advisory created by the Security Working Group to keep the implementation details of a fix for a vulnerability private until the appropriate time to release the fix to the public.
+
+.. _temporary private fork: https://docs.github.com/en/code-security/security-advisories/repository-security-advisories/collaborating-in-a-temporary-private-fork-to-resolve-a-repository-security-vulnerability
+
+.. _security-releases:
+
+Security Releases
+-----------------
+The current process for releasing security fixes involves sending a disclosure and security patch to members of the
+open source security email list and waiting two days before making the patch public.  Instead of this process, we
+propose the following security release process:
+
+#. The maintainer will create an announcement post in the `Security Announcement Section`_ on https://discuss.openedx.org. It should specify the affected repository, the date and time at which the patch will become public, and the severity of the vulnerability it fixes.
```

Comment:
1. Must all patches be announced ahead of time? Is there some minimum warning time that others must be given before making a fix public?
2. Maybe the meaning of "patch will become public" should be spelled out explicitly. It could mean the code becomes public in a merged commit (I'm guessing it is this). It could mean in a public PR (more on this elsewhere). It could mean when it is announced.
3. You may want a timeline for how quickly each of the following steps are implemented. Should all steps be completed within an hour (or less or more?) from merge?

Reply:
Looks good. Thanks!